README.md

```diff
@@ -0,0 +1,35 @@
+# Example EMQX Elixir Plugin Template
+
+An example Mix project that can be used to build an EMQX 5.0.0 plugin.
+
+## Quickstart
+
+1. `make` or `MIX_ENV=prod mix release --overwrite`
+2. Copy `_build/prod/plugrelex/example/example-0.1.0` to the `plugins` directory in your EMQX installation.
```

Comment:
can we make a .tar.gz file instead ?

EMQX 5.0 provides a UI to allow uploading packages from browser and then click "install" button to install.

Reply:
Oops.  The final artifact is actually a `.tar.gz`, but I forgot to add the extension.  :see_no_evil: